### ISSUE TITLE
Icon size from 16px to 24px

### ISSUE THREAD

**Issue by JF-Cozy** [cozy/cozy-ui#1808]
Nos icones par défaut font 16px, contrairement aux icones MUI qui sont en 24px.

Cela pose un souci de cohérence sur des composants dont la taille varie selon la taille de l'icone, comme l'IconButton https://material-ui.com/components/buttons/#icon-buttons qui fait 48px chez MUI (pour une icone de 24px) contre 40px/16px.

⚠️ Penser à supprimer le fix sur l'IconButton au moment de la résolution de cette issue.

**Comment by joel-costa** [comment]
Et j'ajouterai que :
- 1ere solution : C'est ta responsabilité, si tu dois te taper plus d'une semaine à tout corriger tu te démerdes :p
- 2e solution : Aucune idée de l'impact en terme de temps de chargement des apps ou d'autres trucs techniques dont je n'ai pas connaissance mais ça mis à part, c'est celle que je trouve la plus séduisante
- 3e solution : La plus rapide mais on repousse le problème à plus tard ^^

**Comment by Crash--** [comment]
2/ Si on a toutes les icones en 16x16 et 24x24, on va doubler le poids du sprite. Ça me semble pas jouable car quasiment toutes nos app continuent d'utiliser le sprite. (même les nouvelles, cf mespapiers par exemple). C'est peut-être jouable d'imaginer une solution pour que le webpack des apps découpent le sprite pour ne faire qu'un sprite avec ce dont elles ont besoin

**Comment by JF-Cozy** [comment]
@joel-costa aujourd'hui à par la taille de l'iconButton qui n'est pas en 48x48 il n'y a pas de problème plus que ça avec cette histoire d'icone ?

**Comment by joel-costa** [comment]
@JF-Cozy : Nope, de votre côté l'iconButton est le seul truc impacté par ce problème à ma connaissance. C'est surtout chez moi que ça pose problème pour quand je dois designer des icones.

**Comment by JF-Cozy** [comment]
ok j'ouvre une issue l'iconButton : https://github.com/cozy/cozy-ui/issues/2107
